README.md

```diff
@@ -0,0 +1,319 @@
+# Pangea Kong Plugin
+
+The Pangea Kong Plugin is a powerful tool that enhances the functionality of the Kong API Gateway. It provides additional features and capabilities to help you manage and secure your APIs effectively. Pangea Kong Plugin main feature is to pre-process AI request using our [Pangea AI Guard](https://pangea.cloud/services/ai-guard/) service in order to eliminate PII, sensitive data, and malicious content from ingestion pipelines, LLM prompts and responses.
+
+
+## Getting Started
+
+To get started with the Pangea Kong Plugin, follow these steps:
+
+### Prerequisites
+
+1. Make sure you have the following prerequisites installed on your machine:
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- Curl
+
+2. Get a Pangea token and its domain on the Pangea User Console. Token should [have access to][configure-a-pangea-service] the AI Guard service. 
+
+
+### Build
+
+In order to build Pangea Kong Plugin image run:
+
+```bash
+docker build --no-cache . --tag pangea_kong_plugin 
```

Comment:
Why `--no-cache`?

Reply:
It should be build just once, so that we can use `--no-cache` to make sure that we have latest dependencies. But I don't have a strong position, we can remove it if you prefer.